test/checks/color/color-contrast.js

```diff
@@ -906,6 +906,29 @@ describe('color-contrast', function () {
     }
   );
 
+  (shadowSupported ? it : xit)('handles <slot> elements', () => {
+    fixture.innerHTML =
```

Comment:
I couldn't use `shadowCheckSetup` because I needed the final node to be the paragraph, but trying to have the paragraph in the first parameter had the code add the shadow dom to it rather than the container element.